lib/tapioca/cli.rb

```diff
@@ -79,9 +79,14 @@ def todo
       aliases: ["-q"],
       type: :boolean,
       desc: "Supresses file creation output"
+    option :verbose,
```

Comment:
It might be interesting to make this a [`class_option`](https://github.com/Shopify/tapioca/blob/master/lib/tapioca/cli.rb#L36) and [pass it to the generator instance](https://github.com/Shopify/tapioca/blob/master/lib/tapioca/cli.rb#L152-L154) under construction so it can be reused in other commands.